string edge_class
string source_node
string target_node
---
bool resultado